lib/timeline/component/ItemSet.js

```diff
@@ -25,231 +25,232 @@ var BACKGROUND = '__background__'; // reserved group id for background items wit
  * @extends Component
  */
 function ItemSet(body, options) {
-  this.body = body;
```

Comment:
You updated the indent to `tabs`. I like that! Sadly this makes this Pull-Request un-reviewable. Could you maybe change that back?